packages/block-library/src/verse/transforms.js

```diff
@@ -11,14 +11,36 @@ const transforms = {
 			transform: ( attributes ) =>
 				createBlock( 'core/verse', attributes ),
 		},
+		{
```

Comment:
I don't think we need to duplicate the transforms in both blocks, right? Adding them to one of the blocks should be enough?

Reply:
I second what Ella said; considering we're already covering quote to verse above, we should be able to remove this.